### ISSUE TITLE
[DevTools Bug] Could not inspect element with id "15". Error thrown:Cached data for element "15" not found

### ISSUE THREAD

**Issue by santhosh-reddy03** [facebook/react#22479]
### Website or app

https://github.com/santhosh-reddy03/react_practice

### Repro steps

$cd repo
and then start the server using npm start

in browser(mozilla firefox)
when i try to add the user and age, and trying to debug the output in ListUsers component, I m facing this error
![Screenshot from 2021-10-01 18-52-36](https://user-images.githubusercontent.com/53914022/135627139-ca7132b6-f39d-4330-8b7d-0cc5281ab552.png)

### How often does this bug happen?

Only once

### DevTools package (automated)

react-devtools-extensions

### DevTools version (automated)

4.19.0-2178a831a

### Error message (automated)

Could not inspect element with id "15". Error thrown:Cached data for element "15" not found

### Error component stack (automated)

```text
InspectedElementContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:38121:43
Suspense
ErrorBoundary_ErrorBoundary@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:36518:5
div
InspectedElementErrorBoundaryWrapper@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:36967:46
NativeStyleContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:39541:38
div
div
OwnersListContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:34741:37
SettingsModalContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:35182:40
Components_Components@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:41481:52
ErrorBoundary_ErrorBoundary@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:36518:5
div
div
ThemeProvider@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:36635:23
PortaledContent@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:36669:34
div
div
div
ThemeProvider@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:36635:23
SchedulingProfilerContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:42673:45
ProfilerContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:41106:35
TreeContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:29603:31
SettingsContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:30214:35
ModalDialogContextController@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:37030:38
DevTools_DevTools@moz-extension://8a220e8e-c7fe-42e7-9ec5-0b6e67ae5dd5/build/main.js:52130:27
```

### GitHub query string (automated)

```text
https://api.github.com/search/issues?q=Could not inspect element with id . Error thrown:
Cached data for element  not found in:title is:issue is:open is:public label:"Component: Developer Tools" repo:facebook/react
```

**Comment by bvaughn** [comment]
I believe this issue was fixed yesterday by #22472 and we just need to publish a release (today).

**Comment by bvaughn** [comment]
This release will be published as 4.19.1
